core/input/input_event.cpp

```diff
@@ -1041,16 +1041,25 @@ void InputEventAction::_bind_methods() {
 void InputEventGesture::set_position(const Vector2 &p_pos) {
 	pos = p_pos;
 }
+Vector2 InputEventGesture::get_position() const {
```

Comment:
Please make sure to keep one empty line between method definitions for consistency.

Reply:
I've seen that in 3.2 core/os/input_event.cpp, I thought it was a weird coding style.